packages/react/toolbar/src/Toolbar.tsx

```diff
@@ -119,7 +116,8 @@ ToolbarLink.displayName = LINK_NAME;
 const TOGGLE_GROUP_NAME = 'ToolbarToggleGroup';
 
 type ToolbarToggleGroupElement = React.ElementRef<typeof ToggleGroupPrimitive.Root>;
-type ToolbarToggleGroupProps = Radix.ComponentPropsWithoutRef<typeof ToggleGroupPrimitive.Root>;
+type ToggleGroupProps = Radix.ComponentPropsWithoutRef<typeof ToggleGroupPrimitive.Root>;
+type ToolbarToggleGroupProps = ToggleGroupProps;
```

Comment:
Should we be using an interface here?

Reply:
I meant to leave a note on this one. We cannot make this an `interface` because the `ToggleGroupProps` is a union of two interfaces and an interface cannot extend a union.

It will moan at you if you try to so we will know it is extending a union and is safe to be a `type`:

![image](https://user-images.githubusercontent.com/175330/132030120-56296be6-73e9-450e-9b45-fe6918dfe705.png)

I realise it is inconsistent though so is confusing. I'm not sure what else to do here though. Ideally we would figure out generic prop types and then we wouldn't need any discriminated unions for our prop types.

Reply:
Ah I see what you mean. Not much we can do at the moment then 👍